hoomd/hpmc/external/user.py

```diff
@@ -76,7 +85,7 @@ class CPPExternalPotential(ExternalField):
 
             gravity_code = "return r_i.z + box.getL().z/2;"
             gravity = hoomd.hpmc.external.user.CPPExternalPotential(
-                code=gravity_code)
+                code=gravity_code, param_array=[])
```

Comment:
```suggestion
                code=gravity_code)
```
The user should not need to explicitly set `param_array=[]`. Doing so will break all existing scripts that use `CPPExternalPotential).

Also, update this example to run successfully in Sybil.